exporter/signalfxexporter/translation/constants.go

```diff
@@ -431,7 +234,6 @@ translation_rules:
     - device
     - type
 
-
 ## other filesystem metrics
```

Comment:
Can we remove these rules from the exporter?

Reply:
Similar reasoning as above, we need df_complex.used to calculate disk.utilization. We could copy the metric and offload the rest though